{
  "totalNumberSkippingTests": 17,
  "bugType": "passing_passing",
  "totalNumberErroringTests": 0,
  "repo": "google/error-prone",
  "metrics": {
    "FailureNames": [
      "java.lang.AssertionError"
    ],
    "StepsDurationsInSeconds": {
      "CheckoutBuggyBuildSourceCode": 0,
      "ComputeClasspath": 2,
      "ComputeTestDir": 0,
      "GatherTestInformationPreviousBuildSourceCode": 0,
      "ResolveDependency": 11,
      "BuildProjectBuild": 15,
      "BuildProjectPreviousBuildSourceCode": 27,
      "TestProjectBuild": 976,
      "ComputeSourceDir": 0,
      "InitRepoToPush": 6,
      "TestProjectPreviousBuildSourceCode": 688,
      "GatherTestInformationBuild": 0,
      "CloneRepository": 34,
      "CheckoutPatchedBuild": 0,
      "NopolRepair": 28,
      "PushIncriminatedBuild": 6,
      "CommitPatch": 7
    },
    "NbFailingTests": 5,
    "PatchCommitUrl": "http://github.com/google/error-prone/commit/9e2a977ea5078ce2e11fccaf76b980a647a536b8",
    "BugCommitUrl": "http://github.com/google/error-prone/commit/06569874d5fd1b9c34f20c222e39e45e9dca5aa1",
    "NbLibraries": 52,
    "NbFileApp": 1028,
    "NbFileTests": 1428,
    "NbCPU": 56,
    "FreeMemory": 1116203976,
    "TotalMemory": 3153068032,
    "AngelicValuesByTest": {
      "com/google/errorprone/util/ASTHelpersTest": 0,
      "com/google/errorprone/util/ReachabilityTest": 0
    },
    "FreeMemoryByStep": {
      "CheckoutBuggyBuildSourceCode": 1178598744,
      "ComputeClasspath": 1427060696,
      "ComputeTestDir": 1216531120,
      "GatherTestInformationPreviousBuildSourceCode": 1049231504,
      "ResolveDependency": 1157000480,
      "BuildProjectBuild": 1803341616,
      "BuildProjectPreviousBuildSourceCode": 1139633376,
      "TestProjectBuild": 1787153568,
      "ComputeSourceDir": 1229170608,
      "InitRepoToPush": 1431314440,
      "TestProjectPreviousBuildSourceCode": 1087533624,
      "GatherTestInformationBuild": 1767612096,
      "CloneRepository": 1297117552,
      "CheckoutPatchedBuild": 1819533056,
      "NopolRepair": 1918636368,
      "PushIncriminatedBuild": 1268349432,
      "CommitPatch": 1116203976
    },
    "ReproductionDate": "May 11, 2017 4:08:51 PM",
    "BuggyBuildId": 199115933,
    "PatchedBuilId": 199117909,
    "BuggyBuildURL": "http://travis-ci.org/google/error-prone/builds/199115933",
    "PatchedBuildURL": "http://travis-ci.org/google/error-prone/builds/199117909",
    "BuggyBuildDate": "Feb 7, 2017 8:26:08 AM",
    "PatchedBuildDate": "Feb 7, 2017 8:35:54 AM",
    "BugCommit": "06569874d5fd1b9c34f20c222e39e45e9dca5aa1",
    "PatchCommit": "9e2a977ea5078ce2e11fccaf76b980a647a536b8",
    "PatchChangedFiles": 3,
    "PatchAddedLines": 0,
    "PatchDeletedLines": 0,
    "NbRunningTests": 1979
  },
  "bugCommit": "https://github.com/google/error-prone/compare/9e7e37d4320a...06569874d5fd",
  "failingModule": "/root/workspace/google/error-prone/199115933/core",
  "hostname": "spirals-spoon3r",
  "totalNumberFailingTests": 5,
  "error-types": [
    "java.lang.AssertionError"
  ],
  "totalNumberRunningTests": 1979,
  "failing-test-cases": [
    {
      "className": "com.google.errorprone.util.ReachabilityTest",
      "failingMethods": [
        "com.google.errorprone.util.ReachabilityTest#test[17]",
        "com.google.errorprone.util.ReachabilityTest#test[14]",
        "com.google.errorprone.util.ReachabilityTest#test[24]",
        "com.google.errorprone.util.ReachabilityTest#test[12]"
      ],
      "erroringMethods": [],
      "failures": [
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Saw unexpected error on line 9. All errors:\n[/in/Test.java:9: error: [FirstCaseFallsThrough] \n      default:\n      ^\n    (see http://errorprone.info/bugpattern/FirstCaseFallsThrough)]",
          "isError": false
        },
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Saw unexpected error on line 10. All errors:\n[/in/Test.java:10: error: [FirstCaseFallsThrough] \n      default:\n      ^\n    (see http://errorprone.info/bugpattern/FirstCaseFallsThrough)]",
          "isError": false
        },
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Saw unexpected error on line 10. All errors:\n[/in/Test.java:10: error: [FirstCaseFallsThrough] \n      default:\n      ^\n    (see http://errorprone.info/bugpattern/FirstCaseFallsThrough)]",
          "isError": false
        },
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Saw unexpected error on line 16. All errors:\n[/in/Test.java:16: error: [FirstCaseFallsThrough] \n      default:\n      ^\n    (see http://errorprone.info/bugpattern/FirstCaseFallsThrough)]",
          "isError": false
        }
      ],
      "nbFailures": 4,
      "nbErrors": 0
    },
    {
      "className": "com.google.errorprone.util.ASTHelpersTest",
      "failingMethods": [
        "com.google.errorprone.util.ASTHelpersTest#constValue"
      ],
      "erroringMethods": [],
      "failures": [
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Did not see an error on line 5 matching Boolean(false). All errors:\n[/Test.java:3: error: [ConstChecker] Integer(42)\n  static final int A \u003d 42;\n                   ^\n    (see http://errorprone.info/bugpattern/ConstChecker), /Test.java:5: error: [ConstChecker] Integer(0)\n  static final boolean B \u003d false;\n                       ^\n    (see http://errorprone.info/bugpattern/ConstChecker)]",
          "isError": false
        }
      ],
      "nbFailures": 1,
      "nbErrors": 0
    }
  ]
}